tests/test_frame.py

```diff
@@ -880,6 +880,10 @@ def test_types_groupby() -> None:
     s1: pd.Series = df.set_index("col1")["col2"]
     s2: pd.Series = s1.groupby("col1").transform("sum")
 
+    # GH 736
+    check(assert_type(df1.groupby(by="col1").aggregate("size"), pd.Series), pd.Series)
+    check(assert_type(df1.groupby(by="col1").agg("size"), pd.Series), pd.Series)
+
```

Comment:
Can you move these tests to the function `test_types_groupby_agg()` ?

Reply:
I have, what about the other aggregates/transforms in `test_types_groupby` ?

https://github.com/pandas-dev/pandas-stubs/blob/6fe90bbb37128a79068ba41c8ebaaf269ddeab90/tests/test_frame.py#L870-L881

Reply:
If you want, you could add extra tests in another PR, but let's leave this PR to focus on just the one issue.  Thanks.